SIPS/sip-29.md

```diff
@@ -59,13 +59,13 @@ This permission is specified as follows in `snap.manifest.json` files:
 
 ### Snap Implementation
 
-Two methods are defined in the Snap Assets API:
+Four methods are defined in the Snap Assets API:
 
 Any Snap that wishes to provide asset information MUST implement the following API:
 
 #### Get Assets Metadata
 
-`Caip19AssetType` - A string that represents an asset using the [CAIP-19][caip-19] standard.
+`Caip19AssetTypeOrId` - A string that represents an asset using the [CAIP-19][caip-19] standard.
```

Comment:
I don't follow the name change here?

Reply:
On some chains NFTs uses CAIP-19 asset IDs (like EVM) and on some others it's CAIP-19 asset types (like Solana)

Reply:
Wait what? Why doesn't NFTs use caip19 asset IDs?

Reply:
No, it depends on the chain, solana uses asset types :sweat_smile:

Reply:
https://namespaces.chainagnostic.org/solana/caip19#examples